code/modules/power/apc.dm

```diff
@@ -1536,6 +1536,27 @@ ABSTRACT_TYPE(/obj/machinery/power/apc)
 	dir = SOUTH
 	pixel_y = -4
 
+/obj/machinery/power/apc/quark
```

Comment:
This looks like an abstract type and should be marked as such

Reply:
done